sktime/classification/deep_learning/rnn.py

```diff
@@ -11,8 +11,12 @@
 from sktime.classification.deep_learning.base import BaseDeepClassifier
 from sktime.networks.rnn import RNNNetwork
 from sktime.utils.validation._dependencies import _check_dl_dependencies
+from sklearn.utils.warnings import warn
```

Comment:
This does not exist, and should fail with `ModuleNotFoundError`. You need to import it from `sktime`, not `sklearn`.
```suggestion
from sktime.utils.warnings import warn
```